static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, CodeSnippet, Col, Row } from "@canonical/react-components";
+import { CodeSnippetBlockAppearance } from "@canonical/react-components/dist/components/CodeSnippet";
```

Comment:
Is that the recommended way of importing it?

Reply:
Thanks, I rebased on master to get the new react-components and this can be imported from the top level now.